presidio-analyzer/analyzer/analyzer_engine.py

```diff
@@ -17,9 +14,16 @@
 
 class AnalyzerEngine(analyze_pb2_grpc.AnalyzeServiceServicer):
 
-    def __init__(self, registry=RecognizerRegistry(),
-                 nlp_engine=SpacyNlpEngine()):
+    def __init__(self, registry,
```

Comment:
```suggestion
    def __init__(self, registry=None, nlp_engine=None)
```
Otherwise the check `if not nlp_engine` only makes sense if someone explicitly passes None for both arguments